src/main/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSink.java

```diff
@@ -218,22 +218,16 @@ protected void setPersistenceBackend(HDFSBackendImplREST persistenceBackend) {
 
     @Override
     public void configure(Context context) {
-        String cosmosHost = context.getString("cosmos_host");
         String hdfsHost = context.getString("hdfs_host");
```

Comment:
I think now this could be compressed as:

```
String hdfsHost = context.getString("hdfs_host", "localhost");
host = hdfsHost.split(",");
LOGGER.debug("[" + this.getName() + "] Reading configuration (hdfs_host=" + Arrays.toString(host) + ")");
```